aspnetcore/fundamentals/environments.md

```diff
@@ -1,76 +1,100 @@
 
-The profile for IIS Express is shown here:
+[!code-html[Main](environments/sample/WebApp1/Pages/About.cshtml)]
 
-![Project Properties Setting Environment variables](environments/_static/project-properties-debug.png)
+Note: On Windows and macOS, environment variables and values are insensitive. Linux environment variables and values are **case sensitive** by default.
 
-Here is a `launchSettings.json` file that includes profiles for `Development` and `Staging`:
+### Development
 
-[!code-json[Main](../fundamentals/environments/sample/src/Environments/Properties/launchSettings.json?highlight=15,22)]
+The development environment can enable features that should not be exposed in production. For example, the ASP.NET Core templates enable the [developer exception page](xref:fundamentals/error-handling#the-developer-exception-page) in the development environment.
 
-Changes made to project profiles may not take effect until the web server used is restarted (in particular, Kestrel must be restarted before it will detect changes made to its environment).
+The environment for local machine development can be set in the *Properties\launchSettings.json* file of the project. Environment values set in *launchSettings.json* override values set in the system environment.
 
->[!WARNING]
-> Environment variables stored in *launchSettings.json* are not secured in any way and will be part of the source code repository for your project, if you use one. **Never store credentials or other secret data in this file.** If you need a place to store such data, use the *Secret Manager* tool described in [Safe storage of app secrets during development](xref:security/app-secrets).
+The following XML shows three profiles from a *launchSettings.json* file:
 
-### Staging
+[!code-xml[Main](environments/sample/WebApp1/Properties/launchSettings.json?highlight=10,11,18,26)]
 
-By convention, a `Staging` environment is a pre-production environment used for final testing before deployment to production. Ideally, its physical characteristics should mirror that of production, so that any issues that may arise in production occur first in the staging environment, where they can be addressed without impact to users.
+When the application is launched with `dotnet run`, the first profile with `"commandName": "Project"` will be used. The value of `commandName` specifies the web server to launch. `commandName` can be one of :
 
-### Production
+* IIS Express
+* IIS
+* Project (which launches Kestrel)
 
-The `Production` environment is the environment in which the application runs when it is live and being used by end users. This environment should be configured to maximize security, performance, and application robustness. Some common settings that a production environment might have that would differ from development include:
+When an app is launched with `dotnet run`, the hosting environment is displayed.
```

Comment:
Screen shot?

Reply:
Added text output. We like to avoid screenshots except on intro tutorials. Added:

The following output shows an app started with `dotnet run`:
```bash
PS C:\Webs\WebApp1> dotnet run
Using launch settings from C:\Webs\WebApp1\Properties\launchSettings.json...
Hosting environment: Staging
Content root path: C:\Webs\WebApp1
Now listening on: http://localhost:54340
Application started. Press Ctrl+C to shut down.
```